tests/array_interoperability_test.py

```diff
@@ -174,12 +174,25 @@ def testTensorFlowToJaxInt64(self):
   @jtu.sample_product(
     shape=all_shapes,
     dtype=numpy_dtypes,
+    gpu=[False, True],
+    copy=[False, True],
   )
-  def testNumpyToJax(self, shape, dtype):
+  def testNumpyToJax(self, shape, dtype, gpu, copy):
     rng = jtu.rand_default(self.rng())
     x_np = rng(shape, dtype)
-    x_jax = jnp.from_dlpack(x_np)
-    self.assertAllClose(x_np, x_jax)
+    if gpu and jax.default_backend() == "cpu":
+      raise unittest.SkipTest("Skipping GPU test case on CPU")
```

Comment:
This will cause half of test cases to be skipped; it would be better to put this logic in the parameterization
```python
  gpu=[False, True] if jax.test_device_matches(["gpu"]) else [False]
```